index.js

```diff
@@ -0,0 +1,174 @@
+const URL = `https://ada-backtrek-api.herokuapp.com/trips`
+const postURL = `https://ada-backtrek-api.herokuapp.com/trips`
+
+
+let singleTripURL = function(tripID){
+  let singleURL = ``
+  return singleURL+=`https://ada-backtrek-api.herokuapp.com/trips/${tripID}`
+}
+
+let reserveAtrip = function(tripID){
+  let singleURL =``
+  return singleURL+= `https://ada-backtrek-api.herokuapp.com/trips/${tripID}/reservations`
```

Comment:
you set the variable `singleURL` to an empty string, and then return it after concatenating a URL... isn't this the same as
```javascript
let reserveAtrip = function(tripID){
  return `https://ada-backtrek-api.herokuapp.com/trips/${tripID}/reservations`
}
```

Also, camel case on this is a little off, I might call this function `reserveTrip` or `reserveSingleTrip` or something similar